titan-options.php

```diff
@@ -489,44 +495,116 @@ function regala_create_options() {
-            'layout5' => get_template_directory_uri() . '/images/logo.png',
-            'layout6' => get_template_directory_uri() . '/images/site-icon.png',
-         ),
-    ) );    
-  
-    
-	
+         
+    for ( $sectionIndex = 1; $sectionIndex <= 5; $sectionIndex++ ) {
+        $sectionLayout = $titan->createThemeCustomizerSection( array(
+             'name' => sprintf( __( 'Section %d Layout ', 'havana' ), $sectionIndex ),
+             'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+          ) );
+          
+          $sectionLayout->createOption( array(
+              'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+              'id' => 'section' . $sectionIndex . 'layout',
+              'type' => 'radio-image',
+              'options' => array(
+                  'layout1' => get_template_directory_uri() . '/images/header-image-default1-thumb.jpg',
+              ),
+          ) );
+        
+        for ( $boxIndex = 1; $boxIndex <= 4; $boxIndex++ ) {
+                           
+             $sectionBox = $titan->createThemeCustomizerSection( array(
+                 'name' => sprintf(__( 'Section %1$s - Box %2$s Content', 'havana' ), $sectionIndex, $boxIndex ),
+                 'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+              ) );
+              
+              $sectionBox->createOption( array(
+                  'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+                  'id' => 'section' . $sectionIndex . 'box' . $boxIndex . 'post/page',
+                  'type' => 'select',
+                  'options' => array(
+                      '1' => 'Page',
+                      '2' => 'Post',
+                  ),
+                  'default' => '2',
+              ) );
+
+              $sectionBox->createOption( array(
+                  'name' => sprintf(__( 'Background Image', 'havana' ) ),
+                  'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+                  'id' => 'section' . $sectionIndex . 'box' . $boxIndex . 'bg_image',
+                  'type' => 'upload',
+              ) );
+
+              $sectionBox->createOption( array(
+                  'name' => sprintf(__( 'Box Background Color', 'havana' ) ),
+                  'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
+                  'id' => 'section' . $sectionIndex . 'box' . $boxIndex . 'bg_color',
+                  'type' => 'color',
+                  'desc' => __( 'This color set the color of the box content', 'havana' ),
+                  'default' => '#FFFFFF',
+              	//'css' => 'p.tagline-description { color: value }',
+              ) );
+
+              $sectionBox->createOption( array(
+                  'name' => sprintf(__( 'Box Title', 'havana' ) ),
+                  'panel' => sprintf(__('Section %d', 'havana'), $sectionIndex ),
```

Comment:
Ensure proper spacing:

Do this:
```sprintf( __( 'Section %d', 'havana' ), $sectionIndex ),```
instead of:
```sprintf(__('Section %d', 'havana'), $sectionIndex ),```